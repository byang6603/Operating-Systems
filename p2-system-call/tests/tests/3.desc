Verifies detection of crash string at begining of buffer.
