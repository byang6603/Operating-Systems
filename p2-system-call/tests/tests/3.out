XV6_TEST_OUTPUT So we see this!
lapicid 0: panic: XV6_TEST_ERROR CRASH SYSTEM CALL INITIATED. CRASHING SYSTEM.
